src/components/Garden/Filters/ListFilter.js

```diff
@@ -11,22 +12,23 @@ function ListFilter({ items, selected, onChange }) {
           item={item}
           onSelect={onChange}
           selected={selected}
+          helptip={item.toLowerCase()}
         />
       ))}
     </div>
   )
 }
 
-function ListItem({ index, item, onSelect, selected }) {
+function ListItem({ index, item, onSelect, selected, helptip }) {
   const theme = useTheme()
 
+  console.log(item)
   const handleOnClick = useCallback(() => {
     onSelect(index)
   }, [index, onSelect])
 
   return (
     <div
-      onClick={handleOnClick}
       css={`
         margin-bottom: ${1 * GU}px;
         color: ${theme.contentSecondary};
```

Comment:
Let´s update this line to: 
```suggestion
        color: ${theme[selected ? 'content' : 'contentSecondary'};
```

And remove the `${selected === index ? :}` logic below